src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1960,6 +1970,13 @@ bool esbmc_parseoptionst::output_goto_program(
       return true;
     }
 
+    if (cmdline.isset("dump-goto-cfg"))
```

Comment:
Can you explain this option?

Reply:
This already came from a previous PR, I just moved it to be before --goto-functions-*